### ISSUE TITLE
Make owl-carousel dot buttons accessible

### ISSUE THREAD

**Issue by joj0s** [ieee-ihuthess/sb-website#10]
The dot shaped buttons used by owl-carousel to navigate between carousel pages lack any text or information making them an accessibility problem.
